src/workflow/reboot.c

```diff
@@ -26,7 +26,11 @@ static void _reboot(void)
         .value = sectrue_u8,
     };
     upside_down_t upside_down = {
+#if PLATFORM_BITBOX02 == 1
         .value = screen_is_upside_down(),
+#elif PLATFORM_BITBOXBASE == 1
+        .value = false,
```

Comment:
`#else #error` 

or `#if PLATFORM_BITBOXBASE == 1 ... #else ... `

Reply:
ack

Reply:
still missing `#else #error`

Reply:
> or `#if PLATFORM_BITBOXBASE == 1 ... #else ... `

This wins

Reply:
I think I added that, must have force pushed over it